ddtrace/profiling/collector/_lock.py

```diff
@@ -298,7 +298,7 @@ def _maybe_update_self_name(self):
 
             if not self._self_name:
                 self._self_name = ""
-                LOG.warning(
+                LOG.debug(
```

Comment:
cc @taegyunkim since we're making this a debug log, is there any useful info we could put in the debug log?

Reply:
I want some information regarding whether the lock is used in part of some library or stdlib but that would require some more logic here, so nothing at the moment. We'd eventually want this to be exported to instrumentation telemetry logs.